qtox/build_qt_windows.sh

```diff
@@ -15,42 +15,80 @@ parse_arch --dep "qt" --supported "win32 win64" "$@"
 
 "$SCRIPT_DIR/download/download_qt.sh"
 
-OPENSSL_LIBS=$(pkg-config --libs openssl)
-export OPENSSL_LIBS
-
 CROSS_COMPILE="$MINGW_ARCH-w64-mingw32-"
 "${CROSS_COMPILE}gcc" --version
 
-./configure -prefix "$DEP_PREFIX" \
+mkdir qtbase/_build && pushd qtbase/_build
+../configure -prefix "$DEP_PREFIX" \
+  -appstore-compliant \
   -release \
-  -shared \
+  -force-asserts \
+  -qt-doubleconversion \
+  -qt-freetype \
+  -qt-harfbuzz \
+  -qt-libjpeg \
+  -qt-libpng \
+  -qt-pcre \
+  -qt-zlib \
+  -feature-zstd \
+  -no-feature-androiddeployqt \
+  -no-feature-brotli \
+  -no-feature-macdeployqt \
+  -no-feature-printsupport \
+  -no-feature-qmake \
+  -no-feature-sql \
+  -no-feature-dbus \
+  -no-opengl \
   -device-option "CROSS_COMPILE=$CROSS_COMPILE" \
-  -qt-host-path /usr/lib/x86_64-linux-gnu/cmake \
+  -qt-host-path /work/host/qt \
   -platform linux-g++ \
   -xplatform win32-g++ \
-  -openssl "$(pkg-config --cflags --libs openssl)" \
-  -openssl-linked \
```

Comment:
How come `-openssl-linked` was removed?

Reply:
With Qt 6.8.1, I was having issues getting the linked version to find the library at runtime. With runtime-resolved libssl, it works. I'll have another look to see if I can get it working.

Reply:
I've put it back, it works. I think what fixed it was qt-cmake.